addons/website/models/digest.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_website_visitor_cnt = fields.Boolean('Visitors')
+    kpi_website_visitor_cnt_value = fields.Integer(compute='_compute_kpi_website_visitors_cnt_value')
+    kpi_tracked_page_view_cnt = fields.Boolean('Tracked page views')  # Non uniq
+    kpi_tracked_page_view_cnt_value = fields.Integer(compute='_compute_kpi_tracked_page_view_cnt_value')
+
+    def _compute_visitor_cnt(self, ctx):
```

Comment:
Instead of ctx, would probably use real parameters (start_dt, end_dt, website_ids). That way code is easier to understand when looking at main methods and definitions :)

Reply:
Yes indeed, it is changed